sass/components/_results-table.scss

```diff
@@ -1,8 +1,8 @@
-.results-table {
+.f {
```

Comment:
interesting class name.

Reply:
oh no it was a typo, reverted like it was :)